### PR TITLE
Convert AUTHOR and CONTRIBUTE to rst files and add them to the doc

### PR DESCRIPTION
**Pull request by xuhdev**
---

## Checklist:

<details open>

<summary></summary>

- Does this pull request close an issue? We encourage you to open an issue first if this pull request (PR) is not a
  minor change.
  - [ ] No
    - [ ] The change in this pull request is minor.
  - [x] Yes
    - Partly on #126

For the following questions, only check the boxes that are applicable.

- [x] Ensure that the pull request text above the horizontal line is descriptive.
- [x] Add/update relevant tests.
- [x] Add/update relevant documents.

- Do you have triage permission of this repository?
  - [ ] No
    - You are good.
  - [x] Yes
    - Does this pull request close an issue?
      - [ ] No, this pull request also acts as an issue by itself.
        - [ ] Assign yourself.
        - [ ] Label this pull request.
        - [ ] Put a milestone.
        - [ ] Put this pull request under "In Progress" or "Under Review" pipeline on ZenHub.
        - [ ] Put this pull request under the appropriate epic on ZenHub.
        - [ ] Put an estimate on ZenHub.
        - [ ] Add dependency relationship with other issues/pull requests on ZenHub.
      - [x] Yes, this pull request addresses an issue and does not act as an issue.
        - [x] Connect this pull request to the underlying issue on ZenHub.
        - [x] DON'T:
          - DON'T assign yourself or anyone else.
          - DON'T label.
          - DON'T put a milestone.
          - DON'T put this pull request under any epic on ZenHub.
          - DON'T put an estimate on ZenHub.
    - [x] When the pull request is ready, request review.
</details>

### CODE REVIEW

File: AUTHORS.rst [hunk 1]
```diff
@@ -1,6 +1,10 @@
+Authors
+=======
+
 PyDAX is written and maintained by the CODAIT team at International Business Machines Corp. (IBM)
 
 Primary co-maintainers:
 
 * Hong Xu <hongx@ibm.com>
 * Edward Leardi <edward@ibm.com>
+* Daniel Jalova <djalova@us.ibm.com>
```
Comment (xuhdev) [inline AUTHORS.rst:10]: @djalova Does this sound good to you?

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+what the user would actually experience and hopefully would give us more chances to discover bugs.
+
+## Continuous Integration (CI)
+
+We prefer keeping CI configuration files, namely :file:`.travis.yml` and :file:`.appveyor.yml` simple and unscrambled.
+Normally, only test environment, such as Python version, OS and tox environmental variables, or anything that is
+specific to the CI system, such as failure notification. Complicated test dependencies and other test dealings should go
+to :file:`tox.ini` and their respective test files in :file:`tests/`.
+
+## Docs
```
Comment (edwardleardi) [inline CONTRIBUTING.rst]: Missed the subheadings here:
`## Where to Import a Symbol?`, `## Continuous Integration (CI)`, `## Docs`

**Review by edwardleardi — CHANGES_REQUESTED** [review]
We still want to convert the README as well right? We'll do that in a separate PR then?

**Comment by xuhdev** [comment]
> We still want to convert the README as well right? We'll do that in a separate PR then?

Yeah, we'll do it separately. Just to prevent the PR from being too big and too hard to review

File: CONTRIBUTING.rst [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+
+When referencing a symbol that is exposed to a user, in general, we prefer importing the symbol from where the package
+publicly exposes it over importing from where the source code of the symbol is defined, e.g., use ``from .schema import
+SchemaDict`` rather than ``from ._schema import SchemaDict``. This way we have more code paths that would go through
+what the user would actually experience and hopefully would give us more chances to discover bugs.
+
+Continuous Integration (CI)
+---------------------------
+
+We prefer keeping CI configuration files, namely :file:`.travis.yml` and :file:`.appveyor.yml` simple and unscrambled.
```
Comment (edwardleardi) [inline CONTRIBUTING.rst]:
```suggestion
We prefer keeping CI configuration files, namely :file:`.github/workflows/*` and :file:`.gitlab-ci.yml`, simple and unscrambled.
```

**Review by edwardleardi — CHANGES_REQUESTED** [review]
Sorry one other small thing I noticed

**Review by edwardleardi — APPROVED** [review]
LGTM, feel free to merge if you're waiting for Daniel to reply to your comment

File: AUTHORS.rst (hunk 1 above)
Comment (djalova) [reply AUTHORS.rst:10]: Sounds good

**Pull request merged**